### PR TITLE
env variable config for bakeoff

### PR DESCRIPTION
**Pull request by carolinemodic**
Set up the env file for images for m17b. @benadida what are we doing with livecheck?

### CODE REVIEW

**Comment by benadida** [comment]
> Set up the env file for images for m17b. @benadida what are we doing with livecheck?

I'd like to have it as an option, even if I don't know yet if I'm going to demo it.

**Comment by adghayes** [comment]
LGTM. Sidebar: I think we should change the dev dock and zero smartcard pin on `main` because I think it's a risk to have them be defaults.

**Comment by carolinemodic** [comment]
update with livecheck as true

**Comment by mattroe** [comment]
oh nice we added the env variable for printer vs smartcard!

File: .env [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 REACT_APP_VX_SKIP_CVR_ELECTION_HASH_CHECK=FALSE
 REACT_APP_VX_SKIP_SCAN_ELECTION_HASH_CHECK=FALSE
 REACT_APP_VX_SKIP_BALLOT_PACKAGE_AUTHENTICATION=FALSE
 REACT_APP_VX_SKIP_CAST_VOTE_RECORDS_AUTHENTICATION=FALSE
-REACT_APP_VX_DISABLE_CVR_ORIGINAL_SNAPSHOTS=FALSE
-REACT_APP_VX_CONVERTER=ms-sems
+REACT_APP_VX_DISABLE_CVR_ORIGINAL_SNAPSHOTS=TRUE
 REACT_APP_VX_SCREEN_ORIENTATION=portrait
-REACT_APP_VX_PRECINCT_REPORT_DESTINATION=smartcard
+REACT_APP_VX_PRECINCT_REPORT_DESTINATION=laser-sheet-printer
```
Comment (benadida) [inline .env:16]: laser? not thermal?

**Comment by mattroe** [comment]
@adghayes did that so fast I didn't even notice yesterday!

File: .env (hunk 1 above)
Comment (carolinemodic) [reply .env:16]: oh @adghayes what is the right string? you said laser-sheet-printer in the doc

Comment (mattroe) [reply .env:16]: thermal per https://github.com/votingworks/vxsuite/pull/3718

Comment (carolinemodic) [reply .env]: updated to thermal

Comment (adghayes) [reply .env]: oops, thanks for catching my mistake

**Review by benadida — APPROVED** [review]

**Pull request merged**